templates/terraform/encoders/backend_service.go.erb

```diff
@@ -43,10 +43,14 @@ for _, backendRaw := range backends {
 	if !ok {
 		continue
 	}
-	if strings.Contains(backendGroup.(string), "global/networkEndpointGroups") {
-		// Remove `max_utilization` from any backend that belongs to a global NEG. This field
-		// has a default value and causes API validation errors
-		backend["maxUtilization"] = nil
+	serverlessNegMatchers := [2]*regexp.Regexp{regexp.MustCompile("projects/(?P<project>[^/]+)/regions/(?P<region>[^/]+)/networkEndpointGroups/(?P<name>[^/]+)"), regexp.MustCompile("projects/(?P<project>[^/]+)/global/networkEndpointGroups/(?P<name>[^/]+)")}
```

Comment:
regexes only need to match part of the string, so I think just using `(?:global|region/[^/]+)/networkEndpointGroups` as the regex would work too and be a bit less text. any reason to do this way instead?